lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -9,38 +9,52 @@
   <div class="request-response msg msg-confirm copy" id="request-response"></div>
   <ul class="list-fields">
     <li class="field">
-      <label for="id_to">${_("Send to:")}</label><br/>
-      <select id="id_to" name="send_to">
-        <option value="myself">${_("Myself")}</option>
-        %if to_option == "staff":
-        <option value="staff" selected="selected">${_("Staff and admins")}</option>
-        %else:
-        <option value="staff">${_("Staff and admins")}</option>
-          %endif
-          %if to_option == "all":
-            <option value="all" selected="selected">${_("All (students, staff, and admins)")}</option>
-          %else:
-      <option value="all">${_("All (students, staff, and admins)")}</option>
-      %endif
-      </select>
+        ${_("Send to:")}
+        <ul role="group" aria-label="${_('Send to:')}">
```

Comment:
@cahrens I added the `aria-label` because without it, the aXe checker fails me on ["Checkbox inputs with the same name attribute value must be part of a group"](https://dequeuniversity.com/rules/axe/1.1/checkboxgroup).

My thinking was:
- we need either `aria-label` or `aria-labelled-by` for the `role="group"` element, and
- using `aria-labelled-by` requires the use of an element id.

Would that be better? We could wrap the first occurrence of `${_("Send to:")}` in a div with an id, then `aria-labelled-by` to avoid the repetition.

@cptvitamin, any advice here?